bundles/action/org.openhab.action.telegram/README.md

```diff
@@ -15,7 +15,7 @@ As described in the Telegram Bot API, this is the manual procedure needed in ord
 
 - Open a new chat with your new Bot and post a message on the chat
 - Open a browser and invoke `https://api.telegram.org/bot<token>/getUpdates` (where `<token>` is the authentication token previously obtained)
-- Look at the JSON result and write down the value of `result[0].message.chat.id`. That is the chatId. (Note that Telegram group chats chatIds are prefixed with a "-". Include the "-" in the chatId in the config file eg: bot1.chatId: -22334455)
+- Look at the JSON result and write down the value of `result[0].message.chat.id`. That is the chatId. Note that Telegram group chats chatIds are prefixed with a dash "-" that must be included in the config file eg: bot1.chatId: -22334455.

```

Comment:
Excellent!!  One last thing you might have overlooked - please consider, "Note that Telegram group chat..."  (chats -> chat)

Reply:
Additionally, I suggest for readability:

````
config file. (eg. bot1.chatId: -22334455)
````